scripts/check_if_pr_is_low_risk.py

```diff
@@ -109,7 +109,8 @@ def load_diff(
             return [], {}
         elif len(split) == 2:
             diff_files.append((split[1], split[1]))
-        elif len(split) == 3:
+        else:
+            assert len(split) == 3
```

Comment:
This (and any other use of assert) needs a code comment explaining it.

Reply:
Done